earthpy/tests/test_clip.py

```diff
@@ -143,18 +249,20 @@ def test_clip_poly(locs_buff, poly_in_gdf):
     assert len(clipped_poly.geometry) == 3
     assert clipped_poly.geom_type[1] == "Polygon"
 
-
 def test_clip_multipoly(poly_in_gdf, multi_gdf):
     """Test that multi poly returns a value error."""
-    with pytest.raises(ValueError):
-        cl.clip_shp(poly_in_gdf, multi_gdf)
-
+    clip = cl.clip_shp(poly_in_gdf, multi_gdf)
+    assert hasattr(clip, "geometry") and clip.geom_type[0] == "MultiPolygon"
 
-def test_clip_donut_poly(locs_gdf, donut_geom):
-    """Donut holes are multipolygons and should raise ValueErrors."""
-    with pytest.raises(ValueError):
-        cl.clip_shp(locs_gdf, donut_geom)
+def test_clip_multiline(poly_in_gdf, multi_line):
```

Comment:
@nkorinek did you delete the donut_poly test intentionally? perhaps i am missing something and if so please let me know!!

Reply:
Hey! I did delete it on purpose, since the test was checking for multi part polygons, which we are trying to be able to accept right now.

Reply:
perfect.